addons/common/functions/fnc_messageBox.sqf

```diff
@@ -0,0 +1,110 @@
+#include "script_component.hpp"
+/*
+ * Author: mharis001
+ * Opens are message box dialog with the given title and message text.
+ * When an empty picture path is provided, the message width is increased.
+ *
+ * Arguments:
+ * 0: Title <STRING>
+ * 1: Message <STRING>
+ * 2: Confirm <CODE>
+ * 3: Cancel <CODE>
+ * 4: Arguments <ANY> (default: [])
+ * 5: Picture <STRING> (default: "\a3\3den\data\displays\display3denmsgbox\picture_ca.paa")
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * ["Confirm", "Are you sure?", {hint "Confirmed!"}] call zen_common_fnc_messageBox
+ *
+ * Public: No
+ */
+
+if (canSuspend) exitWith {
+    [FUNC(messageBox), _this] call CBA_fnc_directCall;
+};
+
+if (!hasInterface) exitWith {};
+
+params [
+    ["_title", "", [""]],
```

Comment:
Should have a default, let's say:
```suggestion
    ["_title", "Confirmation", [""]],
```

Reply:
I disagree, the default should be empty - to follow with some other displays like this in arma.